tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -615,3 +615,27 @@ foo(
 );
 "
 `;
+
+exports[`test jsx.js 1`] = `
+"<div>
+  {/* comment */
+  }
+</div>;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<div>
+  {/* comment */}
```

Comment:
The comment is no longer missing. It was attached to a JSXText that we do not print in the normal way, but it didn't make sense for it to be printed there. It really should be a dangling comment of the JSXExpression.